dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -129,37 +138,47 @@ public void findAllWithPaginationTest() throws Exception {
                                       .withSubject("ExtraEntry")
                                       .build();
 
-        getClient().perform(get("/api/core/items")
-                   .param("size", "2"))
-                   .andExpect(status().isOk())
-                   .andExpect(jsonPath("$._embedded.items", Matchers.containsInAnyOrder(
-                       ItemMatcher.matchItemWithTitleAndDateIssued(publicItem1, "Public item 1", "2017-10-17"),
-                       ItemMatcher.matchItemWithTitleAndDateIssued(publicItem2, "Public item 2", "2016-02-13")
-                   )))
-                   .andExpect(jsonPath("$._embedded.items", Matchers.not(
-                       Matchers.contains(
-                           ItemMatcher.matchItemWithTitleAndDateIssued(publicItem3, "Public item 3", "2016-02-13")
-                       )
-                   )))
-                   .andExpect(jsonPath("$._links.self.href", Matchers.containsString("/api/core/items")))
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items")
+                                         .param("size", "2"))
```

Comment:
why we have a so large diff due to formatting style? @tdonohue we should improve our code style if different IDE format the code in so different way (making huge the diff) without violate our code style... can you take a look to that or open an issue on JIRA? @tomdesair are you willing to make some cleanup to reduce the changes?

Reply:
During the last rebase I just restored the original formatting which was changed by this PR https://github.com/DSpace/DSpace/pull/2064/files#diff-8bb61e42ed580192f717e4c1f8f94f4dR148